src/Accounts/Accounts/help/Clear-AzContext.md

```diff
@@ -22,13 +22,20 @@ Remove all Azure Credentials, account, and subscription information.
 
 ## EXAMPLES
 
-### Clear global context
-```
+### Example 1: Clear global context
+```powershell
 PS C:\> Clear-AzContext -Scope CurrentUser
 ```
 
 Remove all account, subscription, and credential information for any powershell session.
 
+### Example 2
+```powershell <!-- Aladdin Generated Example --> 
+PS C:> Clear-AzContext -Force  -Scope Process
+```
+
+Remove all Azure credentials, account, and subscription information. (autogenerated)
```

Comment:
`Remove all Azure credentials, account, and subscription information in this powershell session.` maybe better.